rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -63,6 +63,8 @@ vx_size tensor_data_size(RocalTensorDataType data_type) {
             return sizeof(vx_uint32);
         case RocalTensorDataType::INT32:
             return sizeof(vx_int32);
+        case RocalTensorDataType::INT8:
```

Comment:
Has int8 been tested with pybind for audio pipeline?

Reply:
We only use fp32 dtype for audio for now, so no these other dtypes have not been tested with audio pipeline